alvr/xtask/src/build.rs

```diff
@@ -283,6 +283,7 @@ pub fn build_launcher(profile: Profile, reproducible: bool) {
     }
     let common_flags_ref = &common_flags;
 
+    sh.remove_path(afs::launcher_build_dir()).ok();
```

Comment:
is this just the result directory or the all build files directory for the launcher directory (idk I'm confused)
if it's the latter this seems annoying when working on the launcher locally

Reply:
Only result. Build artifacts are in the target folder and then we copy over just the executable

Reply:
The streamers/clients that the launcher downloads end up in the current directory, which is not so good. I understand what you mean, so we need to fix that then make so we delete the folder only with package-launcher and not build-launcher